src/utils.ts

```diff
@@ -0,0 +1,147 @@
+// eslint-disable-next-line max-classes-per-file
+import { IAFile, IAMD, TimelineMoment } from './interfaces';
+
+export class Moment implements TimelineMoment {
+  altText: string;
+
+  desktopImg: string;
+
+  mobileImg: string;
+
+  link: string | undefined;
+
+  constructor() {
+    this.altText = '';
+    this.desktopImg = '';
+    this.mobileImg = '';
+    this.link = undefined;
+  }
+}
+
+export const shuffle = (list: any[]) => {
+  if (!Array.isArray(list)) {
+    throw new TypeError(`Expected an Array, got ${typeof list} instead.`);
+  }
```

Comment:
You don't need this array check since the `T[]` parameter is known to be an array because of the square brackets.